app/views/results/_result_asset.html.erb

```diff
@@ -18,7 +21,7 @@
       <%= image_tag 'medium/processing.gif' %>
     </span>
   <% else %>
-    <%= image_tag(preview_asset_path result.asset) if result.asset.is_image? %>
+    <%= image_tag result.asset.url(:medium) if result.asset.is_image? %>
```

Comment:
Rename _preview_ to _downloadable_image_tag_ in assets_controller.rb. I know you didn't wrote that function, but just to be clear what it does, and why it's not used here.